.github/workflows/ci.yml

```diff
@@ -287,6 +287,7 @@ jobs:
     name: "Remote: Build & test MAPDL ${{ matrix.mapdl-version }} (Extended testing ${{ matrix.extended_testing }})"
     runs-on: ubuntu-latest
     if: github.ref != 'refs/heads/main' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    needs: [smoke-tests]

```

Comment:
Until now we have set the testing after the smoke tests because testing tend to fail a lot and we want to see it early.

I'm Ok with this change though. It might be a good moment to go back to default, since the stability has improved a lot lastly.